static/docs/commands-reference/status.md

```diff
@@ -50,21 +50,45 @@ detected, `dvc status` prints this message:
 ```
 
 This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+be run again by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally a status word is shown describing the
+changes in checksum or status. Given list provides a reference to both, the
+status of the DVC-file and differences in the DVC-files:
 
 - For the local workspace:
-  - _changed_ means the file has changed
+
+  - _changed checksum_ means actual checksum of the files present in workspace
+    doesn't match the one saved in DVC-files;
+  - _always changed_ means that this is a special DVC-file with no dependencies
+    (orphans), which is considered always changed;
```

Comment:
Note that "special DVC-file with no dependencies" is also known as an "orphan" stage file, as mentioned in a few docs:

![image](https://user-images.githubusercontent.com/1477535/60850320-09a89200-a1ac-11e9-837f-4d9ceb17136d.png)

Should we mention that term here? (@shcheklein)